tests/integration/test_issue_2235.py

```diff
@@ -51,14 +38,14 @@ def test_standard_library_is_included(
 with open("{}", 'w') as f:
     f.write(error)
 """.format(
-        output_path
+        built_pex_path, output_path
     )
 
-    script_path = "{}/script.py".format(tmpdir)
+    script_path = os.path.join(tmpdir, "script.py")
     with open(script_path, "w") as f:
         f.write(script)
 
-    subprocess.run([sys.executable, script_path], check=True, env=os.environ)
+    subprocess.run([sys.executable, script_path], check=True)
```

Comment:
Ok, gotcha. You fixed the extraneous env since you directly manipulate sys.path in the script.